MIGRATING_TO_V8.0.0.MD

```diff
@@ -0,0 +1,214 @@
+While before you would receive them as camel case.
+
+Before v8.0.0 some responses used to be camel cased:
+```js
+// Setting the general configuration with a snake cased custom key
+await sdk.configurations.general.update({ 
+  data: {
+      enable_this_feature: true,
+    }
+});
+
+// Retrieving the general configuration with the custom key
+const configuration = await sdk.configurations.general.find();
+
+
+// Before v8.0.0 the custom key would be accessed as
+configuration.data.enableThisFeature
+
+// After v8.0.0 the custom key would be accessed as
+configuration.data.enable_this_feature 
+```
+
+#### Dates
+
+If you are providing custom keys with timestamp in the name, they will now not return as Javascript date objects anymore.
+You will now receive them as what was provided in the request.
+Except if you provide a date object, you will receive an iso string.
+
+Before v8.0.0 some timestamps in responses were returned as Javascript date objects:
+
+```js
+// Setting the general configuration with a snake cased custom key
+await sdk.configurations.general.update({ 
+  data: {
+      creationTimestamp: 0,
+      expiryTimestamp: new Date(2012,0,1),
+      updateTimestamp: '2010-02-05T00:00:00.000Z',
+    }
+});
+
+// Retrieving the general configuration with the custom key
+const configuration = await sdk.configurations.general.find();
+
+
+// Before v8.0.0 the data object in the response would be
+// {
+//     creationTimestamp: new Date(0), 
+//     expiryTimestamp: new Date(2012,0,1), 
+//     updateTimestamp: new Date('2010-02-05T00:00:00.000Z')
+// }
+
+// After v8.0.0 the data object in the response would be
+// {
+//     creationTimestamp: 0,
+//     expiryTimestamp: '2012-01-01T00:00:00.000Z',
+//     updateTimestamp: '2010-02-05T00:00:00.000Z',
+// }
+```
+
+#### Records Affected
+
+If you were providing custom keys with the name `records_affected` or `recordsAffected` they will now return as they are provided.
+While before they would return as `affectedRecords`.
+
+### Reverting to request normalization of custom keys for a single operation
+You may enable normalization of custom keys by setting the option `sdk.service.operation({ normalizeCustomData: true })`
+- Before 8.0.0: `sdk.profiles.update(id, data, options)`
+- After 8.0.0: `sdk.profiles.update(id, data, {...options, normalizeCustomData: true ))`
+
+### Reverting to request normalization of custom keys for all operation a client
+The normalization of custom keys can be reverted to behavior before 8.0.0 using the snippet `sdk = createClient({ ...options, normalizeCustomData: true });`.
+- You may then disable normalization of custom keys for a single operation by  `sdk.service.operation({ normalizeCustomData: false })`
+
+### Affected Properties
+
+These are the affected properties, listed by the relevant SDK method:
+
+- `sdk.configurations.find`
```

Comment:
(minor) Would it make sense to call `sdk` `exh` instead? (also in the examples/snippets above?)